tests/shard1/test_dynamics.py

```diff
@@ -90,14 +90,15 @@ def test_torque_driven(with_contact, with_external_force, cx, rigidbody_dynamics
     ConfigureProblem.initialize(ocp, nlp)
 
     # Test the results
+    time = np.random.rand(1, nlp.ns)
```

Comment:
not useful now but might be tested in the future if time is included into dynamics
aswell to see if time impacts the result when it shouldn't